### PR TITLE
[WIP] Write mode polish

### PR DESCRIPTION
**Pull request by xtine**
- Excerpt toggle label just says "Show"
 - remove show more/less text toggles
- Clean up nesting of comment attachment styles
 - line up "Upload Attachment" and "Save Response" buttons

Fixes: eregs/notice-and-comment#354

screen shot 2016-06-07 at 6 08 24 pm

### CODE REVIEW

**Comment by cmc333333** [comment]
Something's wrong with my local setup, so I can't test locally yet. But! Changes look good in the abstract. Is this in a state to merge?

For future reference, most of the stylesheet changes are just moving things around. New stuff:

* `.comment-context-toggle` -> `a.comment-context-toggle` (with new text color)
* `.comment-controls` has a new nested class
* `.comment-submission`
* `.comment-upload-button`
* `.comment-attachment` margin, `last-child`
* `.comment-submission`

**Comment by xtine** [comment]
Multiple comment attachments are now stacked vertically and also accounts for extra long file names.

screen shot 2016-06-08 at 11 42 52 am

**Comment by xtine** [comment]
Added "Delete response":

screen shot 2016-06-08 at 12 56 01 pm

**Comment by donjo** [comment]
Just confirmed a few things with @jehlers:

- The actual cited text should stay black. The only part that needed to be gray (#767676) was what was in the gray bar.

- The bar should change from "Show" to "Hide" when it's expanded.

**Comment by xtine** [comment]
- Changed the cited text back to black, that's now on eregs/notice-and-comment#361
- Toggled text now Show/Hide
- Added "Delete response" back in.

screen shot 2016-06-08 at 12 56 01 pm

**Comment by donjo** [comment]
Looks good. Is the bug in eregs/notice-and-comment#367 related to this in any way or is it ready to merge?

File: regulations/static/regulations/js/source/views/comment/comment-view.js [hunk 1]
```diff
@@ -116,6 +118,8 @@ var CommentView = Backbone.View.extend({
 
       this.$contextSectionLabel.html(options.label);
 
+      this.$deleteResponseDiv.attr('data-section', options.section);
```
Comment (cmc333333) [inline regulations/static/regulations/js/source/views/comment/comment-view.js]: This works, but we should probably switch to a template at some point

**Comment by xtine** [comment]
The comment delete/save bug wasn't triggered with this PR.

**Comment by cmc333333** [comment]
General todo, we have lots and lots of `e.preventDefault();`'s now, which is concerning. I don't think we need to address that right now, however.

**Pull request merged**